Analysis_of_rich_people.R

```diff
@@ -22,5 +76,33 @@ plot <- ggplot(top_10_billionaires, aes(x = reorder(personName, -finalWorth), y
        fill = "Billionaire") +
   theme(axis.text.x = element_text(angle = 45, hjust = 1))
 
-# Show the plot
+# Display the plot
 print(plot)
+
+# The plot displays ten bars in a barchart. Each bar represents the wealth of 
+# one billionaire. The bar only displays the top 10 richest billionaires.
+# The height of the bar represents the wealth of the billionaire in US-dollars.
+
+# ------------------------------------------------------------------------------
+
+# Next, we decided to formulate a thesis. Based on the dataset, we decided that we want
+# to formulate our thesis about the correlation and causation between certain data 
+# columns and the wealth of the billionaires. Maybe this lets us speculate on certain 
+# circumstances that make it easier or harder for someone to become a billionaire.
```

Comment:
Ich gehe davon aus, dass das viel zu ungenau geschrieben ist... Also wir müssen vermutlich richtig spezifische Thesen reinschreiben... Ich schick dir auf Discord mal unser Data Science Projekt. Das müssen wir wahrscheinlich eher in dem Rahmen machen